api/controllers/v0.1/createPatron.js

```diff
@@ -105,13 +106,13 @@ function createPatron(req, res) {
   }
 
 
-  cardCreatorUsername = cardCreatorUsername ||
+  cardCreatorUsername = process.env.CACHED_CARD_CREATOR_USERNAME ||
     awsDecrypt.decryptKMS(process.env.CARD_CREATOR_USERNAME);
-  cardCreatorPassword = cardCreatorPassword ||
+  cardCreatorPassword = process.env.CACHED_CARD_CREATOR_PASSWORD ||
     awsDecrypt.decryptKMS(process.env.CARD_CREATOR_PASSWORD);
 
   Promise.all([cardCreatorUsername, cardCreatorPassword]).then((values) => {
-    [cardCreatorUsername, cardCreatorPassword] = values;
+    [process.env.CACHED_CARD_CREATOR_USERNAME, process.env.CACHED_CARD_CREATOR_PASSWORD] = values;
```

Comment:
Are these actually environment variables or just variables that you're using for local caching? If they're environment variables, you're okay with overriding them here?

Reply:
@EdwinGuzman I need a better way to cache these per-user rather than at the environment level.  I was definitely wrong in putting them at the process.env level.